Examples/EventSources/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+  name: "EventSources",
+  platforms: [
+    .macOS(.v10_12)
+  ],
```

Comment:
let's see if we can get rid of this? i think this should be fixed when we stop using the `dateformatter` is stuff.